docs/platforms/linux-workstation.md

```diff
@@ -23,3 +23,82 @@ The workstation instance has a cloud volume (or virtual disk) available at `/dat
+
+This TensorFlow project contains a demonstration 4-layer neural network model which runs against the MNIST digits dataset. You can tinker with the ``TensorFlow-2.x_mnist-test.py`` script to setup a different model architecture as you like. To run the default configuration, you can execute the ``run.sh`` script inside the directory, which should produce an evaluation output like below.
+
+```
+[EESSI pilot 2021.12] $ ./run.sh 
+2023-10-09 13:15:04.546828: I tensorflow/core/common_runtime/process_util.cc:146] Creating new thread pool with default inter op setting: 2. Tune using inter_op_parallelism_threads for best performance.
+Epoch 1/5
+1875/1875 [==============================] - 5s 3ms/step - loss: 0.2951 - accuracy: 0.9138
+Epoch 2/5
+1875/1875 [==============================] - 14s 7ms/step - loss: 0.1445 - accuracy: 0.9568
+Epoch 3/5
+1875/1875 [==============================] - 6s 3ms/step - loss: 0.1070 - accuracy: 0.9669
+Epoch 4/5
+1875/1875 [==============================] - 3s 2ms/step - loss: 0.0886 - accuracy: 0.9732
+Epoch 5/5
+1875/1875 [==============================] - 3s 2ms/step - loss: 0.0766 - accuracy: 0.9763
+313/313 - 0s - loss: 0.0745 - accuracy: 0.9774
+
+real    0m41.002s
+user    2m24.151s
+sys     1m54.599s
+```
+Guides on how to utilise EESSI further can be found [here](http://www.eessi.io/docs/using_eessi/eessi_demos/).
+
+
+#### Podman
+
+Podman is a container framework provided in the Workstation appliance for the purpose of installing, managing and developing OCI containers. We **strongly** recommend installing any software you wish to add to the your Workstation via podman, as software installed via the package manager or otherwise located outside of ``/home`` will be removed during image upgrades. In-place upgrades are not supported in the Workstation appliance as to reduce dependancy issues between migrations, instead your ``/home`` directory is kept as a seperate partition and re-mounted after the Workstation has been re-imaged.
+
+The podman is syntatically similar to the Docker CLI. For instance, to setup a basic Apache webserver you can run through the series of commands below:
+
+Pull the Apache image:
+
+```podman pull docker.io/library/httpd```
+
+Query the list locally available images to ensure Apache got pulled correctly:
+
+```podman image ls | grep httpd```
+
+Start the Apache webserver:
+
+```podman run --name apache -p 8080:80 -d httpd```
+
+Query the list of active containers to ensure Apache is running:
+
+```podman ps | grep apache```
+
+Make a request to the Apache webserver to ensure the container is running correctly and is accessible.
+
+```
+ubuntu@workstation-ssh:~$ curl localhost:8080
+<html><body><h1>It works!</h1></body></html>
+```
+
+If setting up the webserver didn't produce any unexpected behaviour, podman is running correctly in your Workstation appliance. For more advanced information about using podman, you can check out the docs [here](https://docs.podman.io/en/latest/).
```

Comment:
I would be really tempted to add:

* building an example container and run it (buildah?)
* using that container will apptainer and podman